src/library_pthread.js

```diff
@@ -465,6 +465,20 @@ var LibraryPThread = {
 
     getNewWorker: function() {
       if (PThread.unusedWorkers.length == 0) {
+#if PROXY_TO_PTHREAD
```

Comment:
This should be `if !PROXY_TO_PTHREAD`, no?

Reply:
Damn :D